full_bear/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.permissions import SAFE_METHODS, BasePermission
+from rest_framework.request import Request
+from rest_framework.views import APIView
+
+
+class IsAdminOrWriteOnly(BasePermission):
+    def has_permission(self, request: Request, view: APIView) -> bool:
+        if request.method in SAFE_METHODS:
+            return bool(request.user and request.user.is_staff)
+        else:
+            return True
```

Comment:
```suggestion
        if request.method not in SAFE_METHODS:
            return False
        return bool(request.user and request.user.is_staff)
```
It's better to exit method as fast as possible

Reply:
Ok :) only one thing. After refactoring `if` statement should return `True`  not `False` :)

Reply:
Ok :) only one thing. After refactoring `if` statement should return `True`  not `False` :)